lib/jobs.rb

```diff
@@ -13,6 +13,7 @@ module Jobs
     "|#{Date.current.year.to_s}"+
     "|#{BLOCKED_PHRASES}"+
     "|(#{SIMPLE_SINGULARS.join('|')})s?"
+  SEARCH_RADIUS = 75
```

Comment:
@peggles2 , you've set the radius as a constant, but you're not passing it to USAJOBS.

Reply:
Yes this is due to master getting merged. Not ready for code review yet. Thanks! :)